src/components/Menu/Menu/Menu.jsx

```diff
@@ -91,12 +95,10 @@ const Menu = forwardRef(
       setIsInitialSelectedState(true);
     }, [setIsInitialSelectedState]);
 
-    useLayoutEffect(() => {
+    useEffect(() => {
```

Comment:
When using `requestAnimationFrame`, long-pressing the arrows didn't work properly with the focus/blur handlers.

Reply:
theoretically, it may cause a very very short delay until the `focus` is actually called ([since `useEffect` is async and `useLayoutEffect` is sync, basically](https://kentcdodds.com/blog/useeffect-vs-uselayouteffect)).
In practice I've played with it quite a bit and didn't see any issue with it.. Do you have specific concerns?